examples/tutorials/translations/hindi/Part 06 - Federated Learning on MNIST using a CNN.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<p>Fix markdown between <code>alice</code> and <code>bob</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2909/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>